resolving/npm-resolver/src/trustChecks.ts

```diff
@@ -52,7 +52,7 @@ export function failIfTrustDowngraded (
       'TRUST_DOWNGRADE',
       `High-risk trust downgrade for "${meta.name}@${version}" (possible package takeover)`,
       {
-        hint: `Earlier versions had ${prettyPrintTrustEvidence(strongestEvidencePriorToRequestedVersion)}, ` +
+        hint: `Earlier released version(${strongestTrustEvidenceVersion}) had ${prettyPrintTrustEvidence(strongestEvidencePriorToRequestedVersion)}, ` +
```

Comment:
Maybe we shouldn't include the specific version number in the message.
Showing something like `semver@7.5.1` while installing `semver@6.3.1` could be confusing.

Thoughts?
```suggestion
        hint: `Previously published releases had ${prettyPrintTrustEvidence(strongestEvidencePriorToRequestedVersion)}, ` +
```

Reply:
I don't think the existing text needs changes. Just add this to the hint:

```
'Trust checks are based solely on publish date, not semver. ' +
          'A package cannot be installed if any earlier-published version had stronger trust evidence.',
```